SeaPublicWebsite/ExternalServices/Models/PropertyTypeEnum.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace SeaPublicWebsite.ExternalServices.Models
```

Comment:
This looks to be specifically for the BRE API with values that match the API spec.
If that is correct then I think this should be called something like `BrePropertyType`, the values should be explicitly set, and there should be a comment explaining that the values must match the BRE API spec.
This probably applies to all the new enums